packages/components/psammead-radio-schedule/README.md

```diff
@@ -0,0 +1,88 @@
+
+## Description
+
+The `RadioSchedule` component is designed to display radio schedule programs in their different states. Currently, the component comprises of a `ProgramCard` which is comprised of a link to the radio-schedule, a state label, brand and episode titles, a summary, and a duration.
+
+## Installation
+
+```jsx
+npm install @bbc/psammead-radio-schedule --save
+```
+
+## Components (WIP)
+
+## ProgramCard
+
+### Props
+
+<!-- prettier-ignore -->
+| Argument | Type | Required | Default | Example |
+| -------- | ---- | -------- | ------- | ------- |
+| brandTitle | string | yes | N/A | `'This is a brand title'` |
+| summary | string | yes | N/A | `'This is a summary'` |
+| episodeTitle | string | yes | N/A | `'29/01/1990'` |
+| duration | string | yes | N/A | `'30:00'` |
+| durationLabel | string | yes | N/A | `'duration'` |
+| state | string | yes | N/A | `'live'` |
+| stateLabel | string | yes | N/A | `'مباشر'` |
+| link | string | yes | N/A | `'https://bbc.com/arabic/articles/c1er5mjnznzo'` |
+| startTime | string | yes | N/A | `13:00` |
+| service | string | yes | N/A | `'news'` |
+| script | object | yes | N/A | `{ canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, }` |
+| dir | string | no | `"ltr"` | `"rtl"` |
+
+### Usage
+
+This component displays a single link as program card with a brand title, an episode title, a summary and a duration.
+
+```jsx
+import React from 'react';
+import { latin } from '@bbc/gel-foundations/scripts';
+import ProgramCard from '@bbc/psammead-radio-schedule/ProgramCard';
+
+<ProgramCard
+  service="news"
+  script={latin}
+  dir="ltr"
+  startTime="13:00"
+  brandTitle="This is a brand title"
+  summary="Could a computer ever create better art than a human?"
+  episodeTitle="29/01/1990"
+  duration="30:00"
+  durationLabel="duration"
+  state="live"
+  stateLabel="Live"
+  link="https://bbc.com/arabic/articles/c1er5mjnznzo"
+/>;
+```
+
+### When to use this component
+
+This component is to be used on the `radio` pages.
```

Comment:
Won't radio schedules be located on front pages?

Reply:
I'm not sure about this, maybe @tochwill can help.

Reply:
Hey - this is going to be on front pages

Reply:
It's also going to be on other pages that the media team are working on...

Reply:
Is there any value in specifying Simorgh pages in the README files?